gcloud-java-dns/src/main/java/com/google/gcloud/dns/ZoneInfo.java

```diff
@@ -255,7 +232,7 @@ public Builder toBuilder() {
     pb.setDescription(this.description());
     pb.setDnsName(this.dnsName());
     if (this.id() != null) {
-      pb.setId(this.id());
+      pb.setId(new BigInteger(this.id()));
```

Comment:
I will leave it as is. I do not see any particular benefit of one solution over another.